### PR TITLE
Better: Add handover protocol doc RD-23015

### PR DESCRIPTION
**Pull request by ndomenec**
This is the pull request to create the Bots api doc on dev portal, see https://jira.ringcentral.com/browse/RD-23015

Here's how it looks:

bots-api-doc-setup

bots-api-doc-setup-2

handover-from-bot-to-agent

handover-from-agent-to-bot

### CODE REVIEW

File: docs/interactions/bots-api.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Bots API
+
+Engage Digital’s Bots API is our way to integrate bots into Engage Digital chat channels. This guide is meant to provide integrators a comprehensive overview of the available features for them to be able to create an implementation.
+
+Currently it's only available on Messenger channels, their documentation is available [here](https://developers.facebook.com/docs/messenger-platform/handover-protocol).
+
+## Setup
+
+Please contact Support to get access to the feature on ED side.
```
Comment (ylecuyer) [inline docs/interactions/bots-api.md:9]:
```suggestion
```

If we publish the doc, we will remove the feature from beta

File: docs/interactions/bots-api.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
+# Bots API
+
+Engage Digital’s Bots API is our way to integrate bots into Engage Digital chat channels. This guide is meant to provide integrators a comprehensive overview of the available features for them to be able to create an implementation.
+
+Currently it's only available on Messenger channels, their documentation is available [here](https://developers.facebook.com/docs/messenger-platform/handover-protocol).
```
Comment (ylecuyer) [inline docs/interactions/bots-api.md:5]:
```suggestion
Currently it's only available on Messenger channels, whose associated documentation on Facebook is available [here](https://developers.facebook.com/docs/messenger-platform/handover-protocol).
```

File: docs/interactions/bots-api.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+## Setup
+
+Please contact Support to get access to the feature on ED side.
+
+You will also need to choose which Facebook app is primary (ED or BOT app), based on that, handover protocol behavior won’t be exactly the same, see FB’s docs for more information.
```
Comment (ylecuyer) [inline docs/interactions/bots-api.md:11]:
```suggestion
You will also need to choose which Facebook app is primary (ED or BOT app), based on that, handover protocol behavior won’t be exactly the same, see aforementioned Facebook documentation for more information.
```

**Comment by ylecuyer** [comment]
Could you also add the swagger specification for the endpoint

File: docs/interactions/bots-api.md (hunk 1 above)
Comment (ndomenec) [reply docs/interactions/bots-api.md:9]: Ok do we have any info on when that will be ?

**Comment by ndomenec** [comment]
Btw I added the workflow part in the doc, I forgot to add it in the first place

File: docs/interactions/bots-api.md (hunk 1 above)
Comment (ylecuyer) [reply docs/interactions/bots-api.md]: will be handled internally

File: specs/engage-digital_openapi3.yaml [hunk 4]
```diff
@@ -341,6 +343,51 @@ paths:
       summary: Getting an attachment from its id
       tags:
         - Attachments
+  '/bots/handover':
+    post:
+      description: This method is used to handover a thread from bot to agent or from agent to bot
```
Comment (ylecuyer) [inline specs/engage-digital_openapi3.yaml:348]: could you add something like, only for messenger, see Bot Api section in documentation

Comment (ndomenec) [reply specs/engage-digital_openapi3.yaml]: Should be better now

**Comment by ndomenec** [comment]
@ylecuyer could you have a last look when you have some time please

**Comment by ylecuyer** [comment]
LGTM

**Comment by ndomenec** [comment]
PR was merged

**Pull request closed without merging**